arches/install/arches-templates/project_name/webpack/webpack-utils/build-image-filepath-lookup.js

```diff
@@ -22,7 +22,7 @@ const _buildImageFilePathLookup = function(publicPath, path, outerAcc, imageDire
 
             return { 
                 ...acc, 
-                [Path.join(publicPath, 'img', filename)]: Path.resolve(__dirname, Path.join(outerPath, subPath))
+                [Path.join(publicPath, 'img', filename).replace(/\\/g, '/')]: Path.resolve(__dirname, Path.join(outerPath, subPath))
```

Comment:
wouldn't this change break non-windows installations?

Reply:
I'm not a Linux expert but my logic here is that Linux paths do not have backslashes and, since this only replaces existing backslashes with forward slashes, it doesn't affect any paths on anything but Windows environments.

If there are case where this assumption about Linux paths doesn't work then this could be changed to include a environment condition to guaranteed only affecting Windows environments.
Ex.
`if(process.platform === 'win32'){`

I did spin up a Ubuntu VM and was able to run the application without issue.